crates/store/re_log_encoding/src/decoder/streaming.rs

```diff
@@ -5,6 +5,7 @@ use tokio::io::{AsyncBufRead, AsyncReadExt as _};
 use tokio_stream::Stream;
 
 use re_build_info::CrateVersion;
+use re_chunk::Span;
```

Comment:
As per my previous comment, this gave me pause. This use of `Span` hasn't directly to do with chunks, so this `use` is confusing.